elide-core/src/main/java/com/yahoo/elide/core/exceptions/InvalidOperatorNegationException.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2016, Yahoo Inc.
```

Comment:
We should update this (and checkstyle if necessary).

Year: 2017
Company: Oath Inc.

We should be adding Oath copyright headers now, right?